R/converters.R

```diff
@@ -20,18 +20,24 @@ int2col <- function(x) {
 #' @title Convert Excel column to integer
 #' @description Converts an Excel column label to an integer.
 #' @param x A character vector
+#' @param allow_null If `TRUE`, will not warn if `NULL` is encountered.
 #' @export
 #' @examples
 #' col2int(LETTERS)
-col2int <- function(x) {
-
+col2int <- function(x, allow_null = FALSE) {
+  if (is.null(x)) {
+    if (allow_null) {
+      return(NULL)
+    }
+    warning("`NULL` was provided to `col2int()`, may cause problem, check input carefully.")
+  }
```

Comment:
I added this for internal debugging. when I was working on `wb_dims()`, when my conditions were still incorrect. when NULL was passed on to col2int, I ended up with messy results.

I added a allow_null to silence warning in case this was expected, 

similar to `allow_na` in assert_class

Reply:
Maybe the function should return `integer(0)`. similar to `as.integer()`. I simply don't like the additional argument just for this option.